src/components/dashboard/DashboardCard.tsx

```diff
@@ -1,18 +1,33 @@
+import Link from 'next/link'
 import React from 'react'
 
 interface DashboardCardProps {
   children?: React.ReactNode
   heading: string
+  href?: string
+  linkText?: string
 }
 
 export default function DashboardCard({
   children,
   heading,
+  href,
+  linkText,
 }: DashboardCardProps) {
   return (
     <div className="rounded-md bg-gray-800 px-10 py-10">
```

Comment:
nit: `px-10` and `py-10` can just be replaced with `p-10`.
```suggestion
    <div className="rounded-md bg-gray-800 p-10">
```